docs/rfc/rfc-1.adoc

```diff
@@ -43,6 +43,34 @@ collects tBTC fees it uses those fees to buy T tokens back for its treasury.
 
 == In Depth
 
+[[group-size]]
+=== Group Size
+There are three main governable parameters that we need to choose:
+* The total size of the group
+* The number of signers required to sign messages
+* The number of group members required for a <<heartbeat,heartbeat>> to be considered "strong".
+
+The total gas cost for sortiton increases linearly with group size by at least
+20k gas per member. Requiring more signers to sign messages makes it harder for
+adversaries to take over the group, but makes it so that the pool is less
+resilient to undelegation or operators going offline. Raising the
+strong-heartbeat threshold decreases the chance that you'll not have enough
+signers to sign transactions, but decreases a wallets lifespan and increases
+overhead.
+
+Another important thing to keep in mind is optics - even though a group size of
+70 with 40 signers required to sign messages might have all of the same
+security properties as a group size of 100 with 51 signers required to sign
+messages, it might not _feel_ as secure. That perception may cause a decreased
+willingness to invest capital in the system leading to less money going over
+the bridge, even though the fees are less overall.
+
+Since we don't have any workable data that allows us to reasonably estimate
+costs (especially with regard to optics), I suggest we start the system out
+at decent https://en.wikipedia.org/wiki/Focal_point_(game_theory)[schelling
+point] of groupsize = 100, signers = 51, strong heartbeat = 70, and then let
+governance adjust from there once we've gathered data from the system being used.
```

Comment:
Not in this PR but we need to figure out how heartbeats actually work cc @eth-r. That is:
- Are we signing every block or every nth block? 
- What happens if one signature failed? Is it enough to fail a heartbeat? Do we retry?
- Is weak heartbeat reported on-chain. If so, who is reporting it? 
- Is failed heartbeat reported on-chain. If so, who is reporting it?
- Is successful heartbeat reported on-chain? Is it every successful one or maybe every nth successful?